README.markdown

```diff
@@ -6548,6 +6548,8 @@ The optional `flags` argument specifies a user flags value associated with the e
 
 When it fails to allocate memory for the current key-value item, then `set` will try removing existing items in the storage according to the Least-Recently Used (LRU) algorithm. Note that, LRU takes priority over expiration time here. If up to tens of existing items have been removed and the storage left is still insufficient (either due to the total capacity limit specified by [lua_shared_dict](#lua_shared_dict) or memory segmentation), then the `err` return value will be `no memory` and `success` will be `false`.
 
+If the sizes of values in the dictionary are not approximate, it is eaisier to encounter `no memory` error because of memory fragmentation. It is recommended to use different dictionaries for different size values. When you encounter `no memory` error, you can also retry a few times to make sure setting successfully.
```

Comment:
If the sizes of values => If the sizes of items.
different size values => different sizes of items.

Reply:
you can also retry a few times to make sure setting successfully
=>
you can also remove more existing items by retrying more times to make the operation successful.